lib/buildFactory.js

```diff
@@ -192,7 +192,12 @@ class BuildFactory extends BaseFactory {
                         container: permutation.image,
                         dockerRegistry: this.dockerRegistry
                     });
-                    modelConfig.environment = permutation.environment;
+
+                    modelConfig.environment = Object.assign({},
+                        modelConfig.environment,
```

Comment:
We should add `[config.environment]` in the js doc for the create function
https://github.com/screwdriver-cd/models/blob/f58bd1a726e78ac46a331fb754fd49642598cf1b/lib/buildFactory.js#L150

Reply:
Added js doc.